input/pagecontent/auereqdi.md

```diff
@@ -1,10 +1,345 @@
 ### AU eRequesting Data for Interoperability in AU eRequesting R1
 
-TBD
+[Australian eRequesting Data for Interoperability (AUeReqDI)](https://sparked.csiro.au/index.php/sparked-products-resources/auereqdi/auereqdi-release-1/) is the product of a national clinician focussed requirements gathering project operating as part of the [Sparked AU FHIR Accelerator](https://sparked.csiro.au/).  The AUeReqDI outputs form a set of data requirements to be considered and referred to as part of the development and definition of AU eRequesting. 
+
+The AUeReqDI builds upon and complements the foundational [Australian Core Data for Interoperability (AUCDI)](https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/) and focuses on the specific use case of eRequesting.  The primary intent of the AUeReqDI is to design and govern a collection of coherent, reusable building blocks known as ‘data groups’. These data groups specify “what” the clinical requirements of the clinical information that should be included for eRequests. However, it does not specify “how” the data is exchanged; this is the role fulfilled by the FHIR standard. AUeReqDI is not required to be implemented as a whole single product. Parts can be implemented as required for specific use cases.
+
+AUeReqDI Release 1 (R1) is focused on an agreement of the minimum data required to support standardised eRequesting within the Australian health context, and forms a common language foundation that allows systems to exchange semantically accurate data for eRequests. It incorporates and builds upon prior work from national and international programs and initiatives, including the Royal College of Pathologists of Australiasia (RCPA)'s Pathology information, Terminology and Units Standardisation (PITUS) framework and the Royal Australian New Zealand College of Radiology (RANZCR's) Radiology Referral Set.  
+
+AU eRequesting is intended to provide an implementable standard for FHIR based interfaces for provider and patient information systems in Australia​ providing:
+- an exchange standard for AUeReqDI (the underpinning clinical data model)
+- Data model and RESTful API interactions that set minimum expectations for a system to send and receive electronic pathology and medical imaging requests in primary and community-based care settings
+- be built on top of for standards development for specific use cases
+
+With AUeReqDI defining clinical data requirements and FHIR AU eRequesting providing detailed FHIR-based profiles the exchange of eRequests, an interpretation of AUeReqDI is necessary which is undertaken through the community.
```

Comment:
Replace with "With AUeReqDI defining clinical data requirements and FHIR AU eRequesting providing the Implementation Guide for FHIR-based electronic requesting of diagnostic imaging and pathology services, an interpretation of AUeReqDI is necessary which is undertaken through the community."

Reply:
Sentence replaced.